internal/signalfx-agent/pkg/monitors/collectd/df/metadata.yaml

```diff
@@ -8,6 +8,8 @@ monitors:
     namespace that the agent is running in for this monitor to be able to
     collect statistics about that filesystem.  This is mostly an issue when
     running the agent in a container.
+    
+    **This plugin is deprecated in favor of the `filesystems` monitor.**
```

Comment:
@atoulme - should we recommend the host metrics receiver instead? That's what we recommend in docs for the filesystems monitor.

Reply:
I'm just following the advice that was already given back when, we can look at deprecating `filesystems` separately next if the hostmetricsreceiver is ready.

Reply:
Ack!